internal/factsengine/gatherers/dir_scan_test.go

```diff
@@ -0,0 +1,154 @@
+package gatherers_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/suite"
+	"github.com/trento-project/agent/internal/factsengine/gatherers"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+)
+
+const dirScanTestBasePath = "/var/test"
+const dirScanTestGlobPattern = "/var/test/*/ASCS*"
+
+type DirScanGathererSuite struct {
+	suite.Suite
+	testFS afero.Fs
+}
+
+func TestDirScanGathererSuite(t *testing.T) {
+	suite.Run(t, new(DirScanGathererSuite))
+}
+
+func (s *DirScanGathererSuite) SetupSuite() {
+	tFs := afero.NewMemMapFs()
+
+	for i := 0; i <= 2; i++ {
+		path := fmt.Sprintf("%s/%d/ASCS%d", dirScanTestBasePath, i, i)
+		_, _ = tFs.Create(path)
+	}
+
+	_, _ = tFs.Create(fmt.Sprintf("%s/1/ASCS3", dirScanTestBasePath))
+	_, _ = tFs.Create(fmt.Sprintf("%s/1/ASDX2", dirScanTestBasePath))
+	_, _ = tFs.Create(fmt.Sprintf("%s/2/ASDX1", dirScanTestBasePath))
+
+	s.testFS = tFs
+}
+
+func (s *DirScanGathererSuite) TestDirScanningErrorDirScaningWithoutGlob() {
```

Comment:
Why `Error` in the name?

Reply:
Still `Error` in the test name, which is a normal test, without raising any error